src/main/java/rs/teslaris/core/model/document/PersonJournalContribution.java

```diff
@@ -29,6 +29,6 @@ public class PersonJournalContribution extends PersonContribution {
     private LocalDate dateTo;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "journal_id", nullable = false)
-    private Journal journal;
+    @JoinColumn(name = "publication_series_id", nullable = false)
```

Comment:
We can keep this change only in the case we rename class to PersonPublicationSeriesContribution

Reply:
I will revert it for now to journal publication.

Reply:
No, it will have to be renamed, because those contributions are included into PublicationSeries class